src/store/actions.js

```diff
@@ -0,0 +1,11 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+export const getNews = createAsyncThunk(
+  'GET_NEWS',
+  async () => {
+    let data;
+    await axios.get('http://localhost:8080/news/').then(res => data = res.data);
+    return data;
+  }
```

Comment:
Rewrite completely.

You don't need to use `then` when you have `await`.

`data` should be a constant.

```javascript
const { data } = await axios({ ... });
return data;
```